From: Brandon Philips <bphilips@suse.de>
Date: Thu, 29 Oct 2009 17:01:49 -0700
Subject: [PATCH 09/56] sky2: fix sky2_link_down copy/paste comment error

commit 809aaaae162e58696c61e1d0c156cfe16bd309e2 upstream.

Fix copy/paste comment error from sky2_link_up to sky2_link_down.

Signed-off-by: Brandon Philips <bphilips@suse.de>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
---
 drivers/net/sky2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/sky2.c b/drivers/net/sky2.c
index 6a10d7b..a76cd66 100644
--- a/drivers/net/sky2.c
+++ b/drivers/net/sky2.c
@@ -1975,7 +1975,7 @@ static void sky2_link_down(struct sky2_port *sky2)
 
 	netif_carrier_off(sky2->netdev);
 
-	/* Turn on link LED */
+	/* Turn off link LED */
 	sky2_write8(hw, SK_REG(port, LNK_LED_REG), LINKLED_OFF);
 
 	if (netif_msg_link(sky2))
-- 
1.7.1

